app/views/comments/_comment.html.erb

```diff
@@ -19,10 +19,10 @@
             </span>
           </a>
           <% if comment.user.twitter_username.present? %>
-            <a href="http://twitter.com/<%= comment.user.twitter_username %>" rel="noopener noreferrer" target="_blank"<%= image_tag("twitter-logo.svg", class:"icon-img") %></a>
+            <a href="https://twitter.com/<%= comment.user.twitter_username %>" rel="noopener noreferrer" target="_blank"<%= image_tag("twitter-logo.svg", class:"icon-img") %></a>
```

Comment:
Think this is on us, but if you could add a `>` between the `"blank"` and `<%= image_tag` to close out the `a` tag that'd be great!

Reply:
Sure!

Reply:
Great catch Andy. HTML is so weird. It's amazing this still seemed to work.